server/plugins/LFUGens.cpp

```diff
@@ -2424,7 +2424,10 @@ static inline bool check_gate_ar(EnvGen* unit, int i, float& prevGate, float*& g
 static inline bool EnvGen_nextSegment(EnvGen* unit, int& counter, double& level) {
     // if (unit->m_stage == ENVGEN_NOT_STARTED) { return true; } // this fixes doneAction 14, but breaks with
     // EnvGen_next_aa
-
+    if (unit->m_stage == ENVGEN_NOT_STARTED) {
```

Comment:
have you investigated the note in the comment above? is there any unwanted behavior here with EnvGen_next_aa?

Reply:
#3671 is a very long thread, with lot of references, this is relevant for the question:
https://github.com/supercollider/supercollider/issues/3094#issuecomment-380844266

Reply:
thanks, glad to know you tested it. please delete the outdated comment.

Reply:
done